Update Cryptomator (#1178)
privacyguides/privacyguides.org: docs/encryption.en.md

    ![Cryptomator logo](assets/img/encryption-software/cryptomator.svg){ align=right }

>     **Cryptomator** is a cloud storage encryption solution designed to solve privacy issues when saving files to cloud storages. It allows you to create vaults that are stored on a virtual drive, in which the contents of the vaults are encrypted and synced with a cloud storage provider.

Author: ```suggestion
    **Cryptomator** is a cloud storage encryption solution designed to solve privacy issues when saving files to the cloud. It allows you to create vaults that are stored on a virtual drive, in which the contents of the vaults are encrypted and synced with a cloud storage provider.
```

        - [:fontawesome-brands-linux: Linux](https://cryptomator.org/downloads)
        - [:pg-flathub: Flatpak](https://flathub.org/apps/details/org.cryptomator.Cryptomator)
        - [:fontawesome-brands-google-play: Google Play](https://play.google.com/store/apps/details?id=org.cryptomator)
        - [:pg-f-droid: F-Droid](https://cryptomator.org/android)
        - [:fontawesome-brands-app-store-ios: App Store](https://apps.apple.com/us/app/cryptomator-2/id1560822163)
        - [:fontawesome-brands-github: Source](https://github.com/cryptomator)

> Cryptomator utilizes the AES encryption standard with a 256 bit key length (AES-256) to encrypt both files and filenames. Cryptomator cannot encrypt access, modification, and creation timestamps nor the number and size of files and folders.

Reviewer: I think this is a "PIN number" moment since the ES in AES stands for Encryption Standard.
```suggestion
Cryptomator utilizes the symmetric-key Advanced Encryption Standard with a 256 bit key length (AES-256) to encrypt both files and filenames. Cryptomator cannot encrypt some metadata such as access, modification, and creation timestamps, nor the number and size of files and folders.
```

Or, since we link to the security architecture details below anyways and I'm not convinced knowing exactly what AES is really contributes to the understanding of the sentence:

```suggestion
Cryptomator utilizes AES-256 encryption to encrypt both files and filenames. Cryptomator cannot encrypt some metadata such as access, modification, and creation timestamps, nor the number and size of files and folders.
```

> Some Cryptomator cryptographic libraries have been [audited](https://community.cryptomator.org/t/has-there-been-a-security-review-audit-of-cryptomator/44) by Cure53. The scope of the audited libraries include: [cryptolib](https://github.com/cryptomator/cryptolib), [cryptofs](https://github.com/cryptomator/cryptofs), [siv-mode](https://github.com/cryptomator/siv-mode) and [cryptomator-objc-cryptor](https://github.com/cryptomator/cryptomator-objc-cryptor). The audit did not extend to [cryptolib-swift](https://github.com/cryptomator/cryptolib-swift), which is a library used on Cryptomator for iOS.
> 
> For more information on Cryptomator and it’s security, visit their documented [security section](https://docs.cryptomator.org/en/latest/security/security-target/).

Reviewer: ```suggestion
Some Cryptomator cryptographic libraries have been [audited](https://community.cryptomator.org/t/has-there-been-a-security-review-audit-of-cryptomator/44) by Cure53. The scope of the audited libraries include: [cryptolib](https://github.com/cryptomator/cryptolib), [cryptofs](https://github.com/cryptomator/cryptofs), [siv-mode](https://github.com/cryptomator/siv-mode) and [cryptomator-objc-cryptor](https://github.com/cryptomator/cryptomator-objc-cryptor). The audit did not extend to [cryptolib-swift](https://github.com/cryptomator/cryptolib-swift), which is a library used by Cryptomator for iOS.

For more information on what Cryptomator does and does not protect against, visit their [security target](https://docs.cryptomator.org/en/latest/security/security-target/) documentation.
```

Author: I'm not sure about this change. The scope of Cryptomator's security documentation is beyond just what it does and does not protect against — that's just the first of three pages. The other two pages detail security architecture, which is quite extensive, and best practices.

> Cryptomator's documentation details it's [security target](https://docs.cryptomator.org/en/latest/security/security-target/), [security architecture](https://docs.cryptomator.org/en/latest/security/architecture/), and [best practices](https://docs.cryptomator.org/en/latest/security/best-practices/).

Reviewer: ```suggestion
Cryptomator's documentation details its intended [security target](https://docs.cryptomator.org/en/latest/security/security-target/), [security architecture](https://docs.cryptomator.org/en/latest/security/architecture/), and [best practices](https://docs.cryptomator.org/en/latest/security/best-practices/) for use in further detail.
```
